numpy/__init__.pyi

```diff
@@ -3159,6 +3159,15 @@ else:
     ]
 
 class integer(number[_NBit1]):  # type: ignore
+    @property
+    def numerator(self: _ScalarType) -> _ScalarType: ...
+    @property
+    def denominator(self) -> L[1]: ...
```

Comment:
I'd argue that `denominator` returning a `builtins.int` instance is somewhat concerning, especially for `np.uint64`.